testing/web3signer_tests/build.rs

```diff
@@ -29,6 +30,8 @@ pub async fn download_binary(dest_dir: PathBuf) {
 
     let version = if let Some(version) = FIXED_VERSION_STRING {
         version.to_string()
+    } else if let Ok(env_version) = env::var("LIGHTHOUSE_WEB3SIGNER_VERSION") {
+        OsString::from(env_version).into_string().unwrap()
```

Comment:
My bad, there's no `OsString` required :sweat_smile: 

```suggestion
        env_version
```

Reply:
nice, done. :)